src/Windows/Avalonia.Win32/TimerHandle.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Avalonia.Win32.Interop;
+using Microsoft.Win32.SafeHandles;
+
+namespace Avalonia.Win32
+{
+    internal class TimerHandle : CriticalHandleZeroOrMinusOneIsInvalid
```

Comment:
Why `CriticalHandle` rather than `SafeHandle`?

Reply:
Ah, that was a mistake. I meant to use `SafeHandleZeroOrMinusOneIsInvalid`, not sure how I ended up with the wrong one. Will fix.